test/f90_correct/novector.f90

```diff
@@ -0,0 +1,25 @@
+! RUN: %flang -S -emit-llvm %s -o - | FileCheck %s -check-prefix=DISABLE-METADATA
+! RUN: %flang -Hx,59,2 -S -emit-llvm %s -o - | FileCheck %s -check-prefix=METADATA
```

Comment:
The FileCheck prefix here is confusing. I suggest renaming it to `IGNORE-DIRECTIVES`.

Reply:
The `VECTOR` one below is confusing too. The test is actually trying to confirm that vector instructions *are not* generated.

Reply:
Done.